### DISCUSSION TITLE
文档导航

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#695] [General]
# 文档导航

该模块将于内容下方生成菜单，用于导航到相同栏目的上一篇和下一篇文档。

https://zh-hans.hbstack.dev/modules/docs/doc-nav/

**Comment by go-alex** [comment]
github.com/hbstack/docs/modules/post-nav 不存在，按照您的库里的应该是doc-nav

**Reply by razonyang** [reply]
感谢反馈，已修正 140e026。
